### ISSUE TITLE
[1.3.x] another class not found - install failed

### ISSUE THREAD

**Issue by Portugao** [Guite/MostGenerator#807]
```
Fatal error: Class 'AbstractMUVideo_Entity_Base_AbstractCollectionTranslation' not found in /domain/modules/MUVideo/lib/MUVideo/Entity/CollectionTranslation.php on line 31
```

**Comment by Portugao** [comment]
Sorry.
Not fixed.
```
Fatal error: Class 'AbstractMUVideo_Entity_Base_AbstractCollectionTranslation' not found in /doamin/modules/MUVideo/lib/MUVideo/Entity/CollectionTranslation.php on line 31
```

**Comment by Guite** [comment]
Thanks. There was another occurance. Fixed in fa2ac040d2940f735b1bef5aa26c0964c7d7b477
